src/main/java/Duke.java

```diff
@@ -1,10 +1,254 @@
+        }
+        try {
+            line = line.substring(6);
+            String description = line.split(" /from ")[0];
+            String eventTime = line.split(" /from ")[1];
+            String startTime = eventTime.split(" /to ")[0];
+            String endTime = eventTime.split(" /to ")[1];
+            Event eventTask = new Event(description, startTime, endTime);
+            addTask(tasks, eventTask);
+        } catch (Exception exception) {
+            System.out.println(exception);
+        }
+    }
+
+    private static void doTodo(String line, ArrayList<Task> tasks) throws DukeException {
+        if (line.length() <= 5 || line.substring(5).isBlank()) {
+            throw new DukeException("☹ OOPS!!! The description of a todo cannot be empty.");
+        }
+        String description = line.substring(5);
+        Todo todoTask = new Todo(description);
+        addTask(tasks, todoTask);
+    }
+
+    private static void doDeadline(String line, ArrayList<Task> tasks) throws DukeException {
+        if (line.length() <= 9 || line.substring(9).isBlank()) {
+            throw new DukeException("☹ OOPS!!! The description of a deadline cannot be empty.");
+        }
+        try {
+            line = line.substring(9);
+            String description = line.split(" /by ")[0];
+            String by = line.split(" /by ")[1];
+            Deadline deadlineTask = new Deadline(description, by);
+            addTask(tasks, deadlineTask);
+        } catch (Exception exception) {
+            System.out.println(exception);
+        }
+    }
+
+    private static void printList(ArrayList<Task> tasks, String[] wordList) throws DukeException {
+        if (wordList.length > 1) {
+            throw new DukeException("☹ OOPS!!! Too much arguments for list");
+        }
+        if (tasks.size() > 0) {
+            System.out.println("Here are the tasks in your list:");
+            for (int i = 0; i < tasks.size(); i += 1) {
+                System.out.printf("%d. %s\n", i + 1, tasks.get(i));
+            }
+        } else {
+            System.out.println("There is no tasks");
+        }
+    }
+
+    private static void doDelete(ArrayList<Task> tasks, String[] wordList) throws DukeException {
+        if (wordList.length != 2) {
+            throw new DukeException("☹ OOPS!!! Wrong number of arguments for delete");
+        }
+        try {
+            int index = Integer.parseInt(wordList[1]) - 1;
+            if (index + 1 > tasks.size()) {
+                throw new DukeException("☹ OOPS!!! index out of bounds");
+            } else {
+                System.out.println("Noted. I've removed this task:");
+                System.out.println(tasks.get(index));
+                tasks.remove(index);
+                System.out.printf("Now you have %d tasks in the list\n", tasks.size());
+            }
+
+        } catch (Exception exception) {
+            System.out.println(exception);
+        }
+        return;
```

Comment:
Consider removing this empty return since it isn't really required. The function will return to the caller once completed.